playwright.config.ts

```diff
@@ -0,0 +1,32 @@
+import { devices, PlaywrightTestConfig } from '@playwright/test';
+
+// Config file docs: https://playwright.dev/docs/test-configuration
+const config: PlaywrightTestConfig = {
+  webServer: {
+    command: 'pnpm start',
+    port: 3000,
+    reuseExistingServer: !process.env.CI,
+  },
+  testIgnore: '**/util/__tests__/**',
+  retries: process.env.CI ? 2 : 0,
+  reporter: process.env.CI
+    ? 'list'
+    : [['html', { outputFolder: 'playwright/report/' }]],
+  outputDir: 'playwright/test-results/',
+  use: {
+    trace: 'on-first-retry',
+    screenshot: 'only-on-failure',
+    video: 'retain-on-failure',
+    testIdAttribute: 'data-test-id',
+  },
+  projects: [
+    {
+      name: 'chromium',
+      use: {
+        ...devices['Desktop Chrome'],
+      },
+    },
+  ],
```

Comment:
This specifies the list of browser projects that will be run. Each project has a name and a set of options.
https://playwright.dev/docs/api/class-testconfig#test-config-projects
https://playwright.dev/docs/api/class-testproject

Reply:
I think the default browser from Playwright is Chromium
https://playwright.dev/docs/browsers#chromium

Reply:
I couldn't find anything in the Playwright documentation that explicitly states that the default browser is Chromium. However, I did find that the default browser type is Chromium in the [Playwright GitHub repo](https://github.com/microsoft/playwright/blob/main/packages/playwright-test/src/index.ts#L62-L63)
```js
  defaultBrowserType: ['chromium', { scope: 'worker', option: true }],
  browserName: [({ defaultBrowserType }, use) => use(defaultBrowserType), { scope: 'worker', option: true }],
```


To confirm this, I locally tested the command `pnpm playwright test --headed` and the video shows that Playwright opened an incognito Chromium window even though the projects config setting was not specified. I also tested the same command in a failing Github Actions workflow and the video shows that a browser was opened and the test did not fail because the projects config setting was deleted.

### Local test video
https://github.com/upleveled/next-js-portfolio-web/assets/80746311/0ef0e7f4-3eb0-42bd-824d-77aa4caff20b

### Failed Github test video
[video.webm](https://github.com/upleveled/next-js-portfolio-web/assets/80746311/78eef894-b89c-48ac-a66c-9b1ed817c9e9)

Reply:
looks like the `projects` config array was removed in some commit